Resources/Variables/flat_dark.css

```diff
@@ -0,0 +1,31 @@
+/* Created by Blackberry Themes */

```

Comment:
Rename this file `flat_dark.css` -> `flat-dark.css`

Reply:
Commits have been pushed. Please check if everything is all right.